editor/store/selectors.js

```diff
@@ -507,6 +507,18 @@ export function getSelectedBlockCount( state ) {
 	return state.blockSelection.start ? 1 : 0;
 }
 
+/**
+ * Returns true if there is a single selected block, or false otherwise.
+ *
+ * @param {Object} state Editor state.
+ *
+ * @return {boolean} Whether a single block is selected.
+ */
+export function hasSelectedBlock( state ) {
```

Comment:
Minor: maybe we should rename the selector to hasOneSelectedBlock, isSingleBlockSelection or something identical to make more obvious that if multiselection is active this returns false.